packages/xod-client/src/utils/normalizeErrcode.js

```diff
@@ -0,0 +1,11 @@
+import * as R from 'ramda';
+
+// :: DataValue -> DataValue
+export default R.pipe(
+  R.when(R.test(/^[eE].*/g), R.tail),
+  x => parseInt(x, 10),
+  R.defaultTo(0),
+  R.min(127),
+  R.max(0),
```

Comment:
It makes possible to bind "E0" to the "error" node. So when it receives a True into "ERR" pin — it does nothing... 🤔